### PR TITLE
Add back Short Form URI, copy legal docs, and various fixes

### PR DESCRIPTION
**Pull request by stevenhartley**
this change also copies over the legal documents and fixes up other parts of the specification.

#83

### CODE REVIEW

File: basics/uri.adoc [hunk 1]
```diff
@@ -104,75 +105,134 @@ a| Validates the UUri object and return `UStatus` with the result of the validat
...
 | `isLongForm(UUri): boolean`
-a| Returns true if the Uri part contains names so that it can be serialized to <<Long Uris>>.
+a| Returns true if the Uri part contains names so that it can be serialized to <<Long URIs>>.
 
 | `isEmpty(UUri): boolean`
 | Returns true if the object contains no information
 
 | `isResolved(UUri): boolean`
-| Returns true if the object contains both names and numbers (ids) such that it can be serialize to both <<Long Uris>> and <<Micro Uris>>
+| Returns true if the object contains both names and numbers (ids) such that it can be serialize to both <<Long URis>> and <<Micro URIs>>
```
Comment (sophokles73) [inline basics/uri.adoc:125]: > ... can be serialized to both ...

File: basics/uri.adoc [hunk 2]
```diff
@@ -104,75 +105,134 @@ a| Validates the UUri object and return `UStatus` with the result of the validat
...
 
 | `isMicroForm(UUri): boolean`
-| Returns true if the Uri part contains the required ids to serialized to Micro format and the fields of the Uri can fit within the specified number of bits and bytes. Both sets of details can be obtained under <<Micro Uris>>
+| Returns true if the Uri part contains the required ids to serialized to Micro format and the fields of the Uri can fit within the specified number of bits and bytes. Both sets of details can be obtained under <<Micro URIs>>
```
Comment (sophokles73) [inline basics/uri.adoc:128]: > ... required IDs to be serialized ...

File: basics/uri.adoc [hunk 3]
```diff
@@ -104,75 +105,134 @@ a| Validates the UUri object and return `UStatus` with the result of the validat
...
+|*`URESOURCE_NAME`* |pchar |The name of the resource. For methods, this is fixed to `rpc`.
+
+|*`URESOURCE_MESSAGE`* |*( pchar / "/" / "?" ) |Protobuf message name as defined in https://developers.google.com/protocol-buffers/docs/overview.
+
+|===
+
+
+*Additional URI Protocol Rules:*
+
+* *MAY* contain schema
```
Comment (sophokles73) [inline basics/uri.adoc:171]: > ... contain scheme

File: basics/uri.adoc [hunk 4]
```diff
@@ -104,75 +105,134 @@ a| Validates the UUri object and return `UStatus` with the result of the validat
...
+* A segment in the path (UE_NAME, UE_VER_MAJOR, URESOURCE_NAME) *MAY* be empty, resulting in two consecutive slashes (//) in the path component, this is equivalent to a wildcard
+* Schema and authority *MUST* be case-insensitive per RFC3986
```
Comment (sophokles73) [inline basics/uri.adoc:173]: > Scheme and ...

File: basics/uri.adoc [hunk 5]
```diff
@@ -104,75 +105,134 @@ a| Validates the UUri object and return `UStatus` with the result of the validat
...
-|*RESOURCE* |pchar |A uThing that can be uniquely identified using a URI and manipulated/controlled by a service. Resources unique scope is when prepended with device and service to become a fully qualified URI. uThings that are not resources may be a service specific features, functionality, capabilities, etc…
-|*MESSAGE* |*( pchar / "/" / "?" ) |Protobuf message name as defined in https://developers.google.com/protocol-buffers/docs/overview.
 |===
 
 
 *Additional URI Protocol Rules:*
 
 * *MAY* contain schema
-* A segment in the path (UE_NAME, UE_VERSION, RESOURCE, METHOD) *MAY* be empty, resulting in two consecutive slashes (//) in the path component, this is equivalent to a wildcard
+* A segment in the path (UE_ID, UE_VER_MAJOR, URESOURCE_ID) *MAY* be empty, resulting in two consecutive slashes (//) in the path component, this is equivalent to a wildcard
```
Comment (sophokles73) [inline basics/uri.adoc:235]: > *MAY* contain scheme

File: basics/uri.adoc (hunk 1 above)
Comment (stevenhartley) [reply basics/uri.adoc:125]: fixed

**Review by PLeVasseur — APPROVED** [review]
LGTM

**Comment by stevenhartley** [comment]
@sophokles73 any objections to merge this?

**Review by sophokles73 — APPROVED** [review]
LGTM

**Pull request merged**